src/ngMock/angular-mocks.js

```diff
@@ -2145,14 +2154,18 @@ function MockHttpExpectation(method, url, data, headers, keys) {
   };
 
   this.params = function(u) {
```

Comment:
Should we give the params here (u, m) better names since we are already refactoring?

Reply:
Suggestions?

Reply:
u -> url
m -> match 

😄

Reply:
`url` is [taken][1] :grin:

[1]: https://github.com/angular/angular.js/pull/16672/files#diff-2a255ed5e9564e25ce6eb711b604f40fR2105

Reply:
What's the difference between url and u anyway?

Reply:
`url` is the URL of the definition (when calling `$httpBackend.when(method, url)`). `u` is the actual URL that `$http` was called with and we want to verify whether it matches the definition.
(Also note, `url` can be a string or a RegExp or a function.)

Reply:
Pretty big difference then! How about requestUrl instead of u?

Reply:
I took a different approach and renamed the `MockHttpExpectation()` arguments to `expectedXyz` (see [eaf7a5db1][1]).
Let me know what you think.

[1]: https://github.com/angular/angular.js/pull/16672/commits/eaf7a5db10b9af09c3ca46997073f4b80c099eff